### PR TITLE
Fix failing cron job

### PR DESCRIPTION
**Pull request by JedMeister**
Closes https://github.com/turnkeylinux/tracker/issues/1962

Also some tidying of the `dehydrated-wrapper` script.

### CODE REVIEW

File: plugins.d/Lets_Encrypt/dehydrated-wrapper [hunk 1]
```diff
@@ -214,23 +218,21 @@ clean_finish() {
 [[ $(which authbind) ]] || fatal "Authbind not installed"
 
 for sig in INT TERM; do
-    trap "clean_finish 1
-    kill -$sig $$" $sig
+    trap 'clean_finish 1 kill -$sig $$' $sig
```
Comment (OnGle) [inline plugins.d/Lets_Encrypt/dehydrated-wrapper:221]: It doesn't look like `clean_finish` takes more than one argument, this doesn't seem right to me.

**Review by OnGle — CHANGES_REQUESTED** [review]
As above I think there's a bug here

File: plugins.d/Lets_Encrypt/dehydrated-wrapper (hunk 1 above)
Comment (JedMeister) [reply plugins.d/Lets_Encrypt/dehydrated-wrapper:221]: I'll double check, but I'm 99% sure that I did test it (with at least ctrl-c) and it seemed to work!?

I'll be back! :)

**Comment by JedMeister** [comment]
As noted above, I have implemented a(n IMO) much nicer way to handle `exit`. :)

See https://github.com/turnkeylinux/confconsole/pull/86/commits/d04cb64566a65ed024b0790751b1bacf059344d2 for the specific changes I've made to this script.

Here is some simplified (pseudo) code demonstrating the gist:
```
#!/bin/bash -e

trap '(exit 130)' INT
trap '(exit 143)' TERM
trap 'clean_exit' EXIT

clean_exit() {
    exit_code=$?
    echo
    if [[ $exit_code -ne 0 ]]; then
        echo "Error cleanup"
    fi
    echo "General cleanup"
    exit $exit_code
}

[...]

if **BAD_CONDITION**; then
    echo "It's all bad!"
    exit 1
fi
echo "All good"
```
Output if `**BAD_CONDITION**`:
```
It's all bad!
Error cleanup
General cleanup
```
Or if all goes well
```
All good
General cleanup
```
Or some other error - inc `SIGINIT`/`SIGTERM`:
```
Error cleanup
General cleanup
```

IMO the beauty is that no special function calls are required. It just captures the exit code of calls to `exit` or the exit code of other events. Here are the "errors" that will be caught and the corresponding exit code of the script:
- SIGINT - exit code 130
- SIGTERM - exit code 143
- code errors (typos/file not found) causing `exit` because `-e` set - exit code 127
- any calls to external commands/built-ins called that return non-zero - propagates exit code
- any calls to `exit` - exit code passed

**Review by OnGle — APPROVED** [review]
Very nice, much more elegant. Looks good to me!

**Review by OnGle — APPROVED** [review]
lgtm!

**Pull request merged**